data/sql/updates/pending_db_world/rev_1567713200326299359.sql

```diff
@@ -0,0 +1,32 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1567713200326299359');
+
+UPDATE `creature_template` SET `flags_extra`= 0 WHERE `entry` IN (26638,31351);
+ 
+UPDATE `creature_template` SET `faction`= 16  WHERE `entry` IN (26620,31339);
+
+DELETE FROM `creature_addon` WHERE `guid` = 127438;
+INSERT INTO `creature_addon` (`guid`, `bytes2`, `emote`) VALUES 
+(127438, 1, 36);
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`= 26623  AND `id` IN (10,11);
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (-127591,-127590,-127589,-127582,-127580,-127579,-127578);
+
+DELETE FROM `creature_formations` WHERE `leaderGUID` IN (127590,27580,127449,127579);
```

Comment:
```sql
DELETE FROM `creature_formations` WHERE `leaderGUID` IN (127590,127580,127449,127579);
```
27580->127580

Reply:
Good catch, thank you! I missed this while reviewing the PR. I have corrected the SQL script, so this is still ready to be merged.